<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>629</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabAssambler">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <attribute name="title">
          <string>Ensamblador</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="pushButtonOpenTxt">
              <property name="minimumSize">
               <size>
                <width>107</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Abrir archivo (.txt)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineDirectionTxt">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QScrollArea" name="scrollArea">
            <property name="minimumSize">
             <size>
              <width>585</width>
              <height>245</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="scrollAreaWidgetContents">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>583</width>
               <height>244</height>
              </rect>
             </property>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="2">
               <widget class="QTextEdit" name="textEditTxt">
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOn</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QTextEdit" name="textEditLineNumber">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                  <horstretch>10</horstretch>
                  <verstretch>10</verstretch>
                 </sizepolicy>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>28</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="palette">
                 <palette>
                  <active>
                   <colorrole role="Text">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>85</green>
                      <blue>127</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Base">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>208</red>
                      <green>208</green>
                      <blue>208</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Window">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>213</red>
                      <green>213</green>
                      <blue>213</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </active>
                  <inactive>
                   <colorrole role="Text">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>0</red>
                      <green>85</green>
                      <blue>127</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Base">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>208</red>
                      <green>208</green>
                      <blue>208</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Window">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>213</red>
                      <green>213</green>
                      <blue>213</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </inactive>
                  <disabled>
                   <colorrole role="Base">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>213</red>
                      <green>213</green>
                      <blue>213</blue>
                     </color>
                    </brush>
                   </colorrole>
                   <colorrole role="Window">
                    <brush brushstyle="SolidPattern">
                     <color alpha="255">
                      <red>213</red>
                      <green>213</green>
                      <blue>213</blue>
                     </color>
                    </brush>
                   </colorrole>
                  </disabled>
                 </palette>
                </property>
                <property name="mouseTracking">
                 <bool>false</bool>
                </property>
                <property name="inputMethodHints">
                 <set>Qt::ImhMultiLine</set>
                </property>
                <property name="frameShape">
                 <enum>QFrame::StyledPanel</enum>
                </property>
                <property name="lineWidth">
                 <number>1</number>
                </property>
                <property name="midLineWidth">
                 <number>1</number>
                </property>
                <property name="verticalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="horizontalScrollBarPolicy">
                 <enum>Qt::ScrollBarAlwaysOff</enum>
                </property>
                <property name="sizeAdjustPolicy">
                 <enum>QAbstractScrollArea::AdjustIgnored</enum>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="textInteractionFlags">
                 <set>Qt::NoTextInteraction</set>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="pushButtonSaveTxt">
              <property name="minimumSize">
               <size>
                <width>78</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Guardar (.txt)</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCreateBin">
              <property name="text">
               <string>Crear binario (.bin)</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabSimulator">
         <attribute name="title">
          <string>Simulador</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QPushButton" name="pushButtonOpenBin">
              <property name="minimumSize">
               <size>
                <width>107</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Abrir archivo (.bin)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineDirectionBin">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QScrollArea" name="scrollArea_2">
              <property name="minimumSize">
               <size>
                <width>491</width>
                <height>212</height>
               </size>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_2">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>489</width>
                 <height>242</height>
                </rect>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QTextEdit" name="textEditLineNumber_bin">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                    <horstretch>10</horstretch>
                    <verstretch>10</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>28</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="palette">
                   <palette>
                    <active>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>85</green>
                        <blue>127</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>208</red>
                        <green>208</green>
                        <blue>208</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>213</red>
                        <green>213</green>
                        <blue>213</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </active>
                    <inactive>
                     <colorrole role="Text">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>0</red>
                        <green>85</green>
                        <blue>127</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>208</red>
                        <green>208</green>
                        <blue>208</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>213</red>
                        <green>213</green>
                        <blue>213</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </inactive>
                    <disabled>
                     <colorrole role="Base">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>213</red>
                        <green>213</green>
                        <blue>213</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Window">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>213</red>
                        <green>213</green>
                        <blue>213</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </disabled>
                   </palette>
                  </property>
                  <property name="frameShape">
                   <enum>QFrame::StyledPanel</enum>
                  </property>
                  <property name="lineWidth">
                   <number>1</number>
                  </property>
                  <property name="midLineWidth">
                   <number>1</number>
                  </property>
                  <property name="verticalScrollBarPolicy">
                   <enum>Qt::ScrollBarAlwaysOff</enum>
                  </property>
                  <property name="horizontalScrollBarPolicy">
                   <enum>Qt::ScrollBarAlwaysOff</enum>
                  </property>
                  <property name="sizeAdjustPolicy">
                   <enum>QAbstractScrollArea::AdjustIgnored</enum>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                  <property name="textInteractionFlags">
                   <set>Qt::NoTextInteraction</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="textEditBin">
                  <property name="enabled">
                   <bool>true</bool>
                  </property>
                  <property name="palette">
                   <palette>
                    <active>
                     <colorrole role="Highlight">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>160</red>
                        <green>160</green>
                        <blue>160</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Link">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>160</red>
                        <green>160</green>
                        <blue>160</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </active>
                    <inactive>
                     <colorrole role="Highlight">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>160</red>
                        <green>160</green>
                        <blue>160</blue>
                       </color>
                      </brush>
                     </colorrole>
                     <colorrole role="Link">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>160</red>
                        <green>160</green>
                        <blue>160</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </inactive>
                    <disabled>
                     <colorrole role="Link">
                      <brush brushstyle="SolidPattern">
                       <color alpha="255">
                        <red>160</red>
                        <green>160</green>
                        <blue>160</blue>
                       </color>
                      </brush>
                     </colorrole>
                    </disabled>
                   </palette>
                  </property>
                  <property name="verticalScrollBarPolicy">
                   <enum>Qt::ScrollBarAlwaysOn</enum>
                  </property>
                  <property name="horizontalScrollBarPolicy">
                   <enum>Qt::ScrollBarAlwaysOff</enum>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="spacing">
                 <number>1</number>
                </property>
                <item>
                 <widget class="QLabel" name="labelOpcode">
                  <property name="maximumSize">
                   <size>
                    <width>53</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Opcode:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditOpcode">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>81</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>81</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="spacing">
                 <number>1</number>
                </property>
                <item>
                 <widget class="QLabel" name="labelOperando">
                  <property name="text">
                   <string>Operando:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditOperando">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>81</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>81</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>1</number>
                </property>
                <item>
                 <widget class="QLabel" name="labelAC">
                  <property name="maximumSize">
                   <size>
                    <width>53</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>LCD</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditLCD">
                  <property name="minimumSize">
                   <size>
                    <width>81</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>81</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="spacing">
                 <number>1</number>
                </property>
                <item>
                 <widget class="QLabel" name="labelAC_2">
                  <property name="maximumSize">
                   <size>
                    <width>53</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>AC:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEditAC">
                  <property name="minimumSize">
                   <size>
                    <width>81</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>81</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="readOnly">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="pushButtonSimulate">
              <property name="minimumSize">
               <size>
                <width>78</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Simular</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonStep">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Step</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonReset">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Reset</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QRadioButton" name="radioButtonDebugMode">
              <property name="text">
               <string>Debug mode</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>pushButtonOpenTxt</tabstop>
  <tabstop>lineDirectionTxt</tabstop>
  <tabstop>pushButtonCreateBin</tabstop>
  <tabstop>pushButtonOpenBin</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>pushButtonSimulate</tabstop>
  <tabstop>pushButtonStep</tabstop>
  <tabstop>pushButtonReset</tabstop>
  <tabstop>radioButtonDebugMode</tabstop>
  <tabstop>lineEditOpcode</tabstop>
  <tabstop>lineEditOperando</tabstop>
  <tabstop>lineEditLCD</tabstop>
  <tabstop>lineDirectionBin</tabstop>
  <tabstop>scrollArea_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
